### ISSUE TITLE
HTTP 500 error on 'latest' image

### ISSUE THREAD

**Issue by pietrygamat** [BeryJu/oauth1-test-server#116]
The `latest` image that is published into ghcr does not seem to work as expected - the application crashes on parsing `Authorization` header from request sent to `/oauth/request_token`. 

When pulling image version `v1.1`, the same request is successful. I tried to rebuild the image from a few commits, but it seems the problems stats as soon as 9fab57e4364afd92de0bd3f915b7b280ff2c89a0 (first commit afer `v1.1`).

**Comment by BeryJu** [comment]
Yeah I'm aware of the error, hence [here](https://github.com/goauthentik/authentik/blob/main/tests/e2e/test_source_oauth_oauth1.py#L62) is still pinned to v1.1, I haven't yet had the time to look into this to figure out what exactly is broken

**Comment by pietrygamat** [comment]
I would suggest updating the example in the README in the meantime.
```
services:
  oauth1-test-server:
    image: ghcr.io/beryju/oauth1-test-server:v1.1
```
I find this project really useful, but getting hit by cryptic errors in a language I am not familiar is disheartening.  A simple example update solves the problems for others like me.

**Comment by BeryJu** [comment]
yeah good point
